libs/translator/reader/read_shader.cpp

```diff
@@ -232,7 +232,8 @@ AtNode* UsdArnoldReadShader::Read(const UsdPrim &prim, UsdArnoldReaderContext &c
     bool isArnoldShader = (TfStringStartsWith(id.GetString(), str::t_arnold_prefix.GetString()));
     const AtNodeEntry* nentry = isArnoldShader ? AiNodeEntryLookUp(id.GetString().substr(7).c_str()) : nullptr;
 
-    bool isUvTexture = (id == str::t_UsdUVTexture);    
+    bool isTextureShader = (id == str::t_UsdUVTexture || id == str::t_arnold_image);
```

Comment:
Would we need to do any special consideration for MaterialX images/textures?

Reply:
good point. I created #1881 for that